smf/switch_zone_setup/switch_zone_setup

```diff
@@ -37,8 +37,4 @@ for i in "${!USERS[@]}"; do
     fi
 done
 
-# enable ipv6 forwarding and routing
-pfexec routeadm -e ipv6-forwarding -u
-pfexec routeadm -e ipv6-routing -u
```

Comment:
Just double checking - when we were checking a system live last week, @rcgoodfellow had us disable `ipv6-routing` but _not_ `ipv6-forwarding`. We do want to disable both, though?

Reply:
I believe disabling both is the correct thing to do in the switch zone. But we'll definitely want laps on dogfood. However, it is critically important that the GZ have `ipv6-forwarding` enabled.